web-registry/src/components/organisms/ProjectTopSection.tsx

```diff
@@ -490,6 +497,30 @@ function ProjectTopSection({
               </Description>
             </div>
           )}
+          {batchData?.totals && (
+            <Box
+              sx={{
+                display: 'flex',
+                gap: 2,
+                justifyContent: 'space-between',
+                mt: { xs: 10, sm: 12, md: 16 },
+                mb: { xs: 10, sm: 12, md: 0 },
+              }}
+            >
+              <LabeledNumber
+                label="Credits Available"
```

Comment:
Why not using "Credits tradable" to be consistent with naming on regen ledger and with the ecocredits table? cc/ @erikalogie